### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by JAStanton** [zalmoxisus/remote-redux-devtools#32]

**Comment by zalmoxisus** [comment]
Thanks for the suggestion, but not sure I understand what you mean by editing and deleting. Do you mean only visually or delete the objects from the app's store? If thirst, then it should be implemented on the monitors part ([`redux-devtools-log-monitor`](https://github.com/gaearon/redux-devtools-log-monitor) or [`redux-devtools-inspector`](https://github.com/alexkuz/redux-devtools-inspector)).

**Comment by JAStanton** [comment]
So say I have an entities store, with a user as one of the keys. I have various actions to fetch user from my server, update user name etc... It would be great, while visually inspecting the user node in my store to right click and delete the user. Today I would have to write a reducer and fire an action to delete it. But just to quickly test the behavior of dropping the user I could just do it from the inspector.

Additionally this would help when testing if I have a reducer that spits out bad data that puts my app in a funky state, I want to be able to just right click and change the values from the visual inspector so that I can put the app in a good state, fix my reducer and test again without having to blow away the entire store and re-initializing everything.

I'll look into the 2 things you linked me to see if any of them have those abilities

**Comment by zalmoxisus** [comment]
I see. It's a nice suggestion, but in Redux [the state is read-only](http://redux.js.org/docs/introduction/ThreePrinciples.html) and can be changed only with reducers.

**Comment by JAStanton** [comment]
Yeah I understand, there would have to be a global reducers for common operations like delete, duplicate, edit etc... that would be generic enough that by right clicking a node visually and selecting `delete` It would fire an action with context of what node you selected to the generic `delete` reducer for instance.

It's totally do-able, I was just hoping someone already did it :P

**Comment by zalmoxisus** [comment]
Well, in case you or someone else wants to implement this, here's the path to follow:

1. Add the ability to edit/add/remove in [`react-json-tree](https://github.com/alexkuz/react-json-tree). Also an `onChange` property, which monitors will use.
2. Implement it in the monitors ([`redux-devtools-log-monitor`](https://github.com/gaearon/redux-devtools-log-monitor) or [`redux-devtools-inspector`](https://github.com/alexkuz/redux-devtools-inspector)): use a method for the property above, which would dispatch `IMPORT_STATE`. Obviously, only the last action can be edited.
